product_supplierinfo_for_customer/views/product_views.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2015 AvanzOSC
+     Copyright 2015-18 Tecnativa
+     Copyright 2017-19 ForgeFlow
+     Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="product_customerinfo_form_view" model="ir.ui.view">
+        <field name="name">product.customerinfo.form.view</field>
+        <field name="model">product.customerinfo</field>
+        <field name="arch" type="xml">
+            <form string="Customer Information">
+                <group>
+                    <group string="Customer">
+                        <field name="company_id" invisible="1" />
+                        <field
+                            name="product_id"
+                            groups="product.group_product_variant,base.group_multi_company"
```

Comment:
```suggestion
                            groups="product.group_product_variant"
```
with the above `<field name="company_id" invisible="1" />`, you can remove these